APMrover2/mode_steering.cpp

```diff
@@ -1,6 +1,12 @@
 #include "mode.h"
 #include "Rover.h"
 
+bool ModeSteering::_enter()
+{
+    g2.motors.slew_limit_motors(true);
+    return true;
+}
+
 void ModeSteering::update()
```

Comment:
We should be controlling the slew rate in all modes of Rover - not just Steering.  Can this PR be updated to include all modes?